fpdf/template.py

```diff
@@ -147,6 +147,29 @@ def _parse_multiline(s):
             return False
         return None
 
+    def parse_json(self, infile, encoding="utf-8"):
+        """
+        Load the template definition from a JSON file.
+        The data must be structured as an array of objects, with names and values exactly
+        equivalent to what would get supplied to load_elements(),
+
+        Arguments:
+
+            infile (string): The filename of the CSV file.
```

Comment:
```suggestion
            infile (string): The filepath of the CSV file.
```

Reply:
I copied that from `parse_csv()` (and very obviously so, since I forgot to change the file type...)

I'll change it to "filepath" in both places, since that is indeed more clear.